src/cbc_sdk/base.py

```diff
@@ -910,7 +910,7 @@ def _refresh_if_needed(self, request_ret):
                 message = request_ret.text
 
             raise ServerError(request_ret.status_code, message,
-                              result="Did not update {} record.".format(self.__class__.__name__))
+                              result="Did not update {} record.".format(self.__class__.__name__), uri=None)
```

Comment:
Do we need to explicitly provide uri if it is None?

Reply:
I suppose not, but it's useful to document the places where we have no `uri`, to potentially revisit those locations if, in future, we can provide a URI.

Reply:
I think you could use `request_ret.url` here?